tests/apps/core/test_language_chooser.py

```diff
@@ -58,6 +61,8 @@ def test_language_chooser_available_language_with_translated_page(self):
                 'topbar__menu__list__item--en">'
             ),
         )
+        self.assertContains(response, "en, switch to Anglais")
```

Comment:
This will break once the translations become available right? Why not include the translations in this PR and make the correct test directly?

Reply:
Done